presto-docs/src/main/sphinx/optimizer/logical-planner.rst

```diff
@@ -0,0 +1,57 @@
+*****************
+Logical Planner
+*****************
+
+Presto currently has a rule based optimizer. A given SQL query goes through syntactic
```

Comment:
I think @KBP-TDC  was saying that even talking about all these different stages was more low level than users need.   Maybe he can give a more specific description of what he's looking for or if  this is good.